core/src/main/java/bisq/core/dao/DaoFacade.java

```diff
@@ -358,16 +360,92 @@ public void publishBlindVote(Coin stake, ResultHandler resultHandler, ExceptionH
     // Use case: Presentation of phases
     ///////////////////////////////////////////////////////////////////////////////////////////
 
-    public int getFirstBlockOfPhase(int height, DaoPhase.Phase phase) {
-        return periodService.getFirstBlockOfPhase(height, phase);
-    }
-
-    public int getLastBlockOfPhase(int height, DaoPhase.Phase phase) {
-        return periodService.getLastBlockOfPhase(height, phase);
-    }
-
-    public int getDurationForPhase(DaoPhase.Phase phase) {
-        return periodService.getDurationForPhase(phase, daoStateService.getChainHeight());
+    // Because last block in request and voting phases must not be used fo making a tx as it will get confirmed in the
+    // next block which would be already the next phase we hide that last block to the user and add it to the break.
+    public int getFirstBlockOfPhaseForDisplay(int height, DaoPhase.Phase phase) {
+        int firstBlock = periodService.getFirstBlockOfPhase(height, phase);
+        switch (phase) {
+            case UNDEFINED:
+                break;
+            case PROPOSAL:
+                break;
+            case BREAK1:
+                firstBlock++;
```

Comment:
This seems like the opposite of what the comment says. This will show the second block of the break as the first block.

Reply:
Yes, should rename the var or refactor it so its less hacky...